rocklibrary/rocklibrary_endpoint/recieve_image_endpoint.py

```diff
@@ -33,11 +33,23 @@ def getImageUploadAndColor():
         print(request.get_header(head))
         if headers.get(head) not in request.get_header(head):
             abort(406, 'Incorrect headers')
+    
+    colors = ['green', 'blue', 'yellow', 'red', 'purple', 'pink', 'black', 'brown', 'dark red', 'dark yellow', 'dark blue',
+    'light red', 'light green', 'grey', 'orange', 'white']
     colorRequest = request.forms.get('color')
     imageRequest = request.files.get('image')
-    if not (colorRequest or imageRequest):
+    userHeightRequest = request.forms.get('userHeight')
+    wallHeightRequest = request.forms.get('wallHeight')
```

Comment:
These variables should be a conditional so turn it into an if statement and don't throw anything if it doesn't exist. Instead if it does not exist make them an empty string